### PR TITLE
Add EIP: Ethereum Mixing Transaction Supervision

### PR DESCRIPTION
**Pull request by Evelynlhn**
**ATTENTION: ERC-RELATED PULL REQUESTS NOW OCCUR IN [ETHEREUM/ERCS](https://github.com/ethereum/ercs)**

--

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by abcoathup** [comment]
@Evelynlhn this appears to be an application layer standard for monitoring transactions rather than at the Ethereum protocol level.  Application layer standards are ERCs.  If this is the case then please close this PR and create a new PR in the ERC repo: https://github.com/ethereum/ercs

Please note EIP/RIP/ERC numbers are issued by editors and associates.  You don't get to pick your own.

File: EIPS/eip-8888.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+eip: 8888
```
Comment (abcoathup) [inline EIPS/eip-8888.md:2]:
```suggestion
eip: to be assigned
```

Comment (Evelynlhn) [reply EIPS/eip-8888.md:2]: Thank you very much for your guidance. I acknowledge my limited experience, and I will carefully review my documents again.

**Comment by abcoathup** [comment]
@Evelynlhn you can close this PR.  A PR has been created in the ERCs repo and assigned an ERC number.
https://github.com/ethereum/ERCs/pull/218

**Comment by g11tech** [comment]
> @Evelynlhn you can close this PR. A PR has been created in the ERCs repo and assigned an ERC number. [ethereum/ERCs#218](https://github.com/ethereum/ERCs/pull/218)

closing for the reason above

**Pull request closed without merging**